src/States/MapViewStateHelper.cpp

```diff
@@ -187,8 +187,9 @@ bool validLanderSite(Tile* t)
 	}
 
 	// bleh, direct copy from Tile::distanceTo()
-	int _x = t->x() - ccLocationX(), _y = t->y() - ccLocationY();
-	float _dist = sqrt((_x * _x) + (_y * _y));
+	int _x = t->x() - ccLocationX();
+	int _y = t->y() - ccLocationY();
```

Comment:
Technically this is sneaking in an unrelated change. This should probably have been done in a separate commit. No issue with the change though.